dependency_support/com_github_facebook_zstd/decodecorpus.patch

```diff
@@ -0,0 +1,172 @@
+diff --git tests/decodecorpus.c tests/decodecorpus.c
```

Comment:
we we already have a pull request against https://github.com/facebook/zstd for landing that patch upstream? This will significantly help landing this internally.

Reply:
alternativly we could explore adding a subset of golden test frame generated from decodecorpus to this PR that cover an equivalent subset of the current decoding functionalities.

Reply:
created https://github.com/facebook/zstd/pull/4102 w/ your patch and appropriate attribution.

Reply:
Thanks! How should we now proceed with the `zstd` dependency after https://github.com/facebook/zstd/pull/4102 was merged? It will take some time for the maintainers to publish a new release that will include modifications to the `decodecorpus`. Should we import here `zstd` at https://github.com/facebook/zstd/commit/fdfb2aff39dc498372d8c9e5f2330b692fea9794 or should we still use the patch?

Reply:
I think we'd be fine to depend on a non release version `fdfb2aff39dc498372d8c9e5f2330b692fea9794`.

Reply:
Bumped `zstd` dependency to `fdfb2aff`